src/test/java/uk/gov/dhsc/htbhf/steps/AddressLookupSteps.java

```diff
@@ -128,6 +128,13 @@ public void assertPostcodeInWrongFormat() {
         assertPostcodeErrorPresent("Enter a correct postcode, like AA1 1AA");
     }
 
+    @Then("^I see the postcode: (.*) in the input in the same format I entered it in")
```

Comment:
The wording on this isn't completely clear. How about `^The postcode remains in the same format as I entered it: (.*)$` or similar?

Reply:
Replaced with 'I see the postcode: <postcode> in the same format I entered it in'